### DISCUSSION TITLE
Add Nginx-Proxy-Manager

### DISCUSSION THREAD

**Discussion by dr-diffie** [coollabsio/coolify#473] [Feature Requests]
https://github.com/NginxProxyManager/nginx-proxy-manager

A little awesome Open-source application which allows Users/Powersuser to set an configure everything around an Nginx proxy. 

This includes also a few awesome settings for setting up a ReverseProxy which could also be integrated into Coolify and therefore increasing the Security of the Instance and Serviced hosted on it. 

Thanks Andreas and all contributors for this awesome tool. 

Could not think of a life without coolify anylonger. 🙏🏻

**Comment by gregfr** [comment]
I agree, I'm using NPM it's easier than Traefik and in most case it's enough.

**Comment by AlejandroAkbal** [comment]
I miss the included rate limit and cache options of Nginx 

Traefik and Caddy don't have them by default and you have to go through the rabbit hole of installing third party extensions for such a basic requirement 😭

I hope Nginx is integrated soon to Coolify
